string[] name

float64[] link_position
float64[] motor_position
float64[] link_velocity
float64[] motor_velocity
float64[] effort
float64[] temperature

float64[] position_reference
float64[] velocity_reference
float64[] effort_reference
float64[] stiffness
float64[] damping

float64[] fault

string aux_name
float64[] aux